DeployPostScript.ps1

```diff
@@ -0,0 +1,74 @@
+# Gets Name of all Jobs
+$AllJobs = Get-VBRJob -ea silentlyContinue -WarningAction silentlyContinue | Select-Object -ExpandProperty Name
+
+# Make sure we actually have jobs we can run this on.
+if ([string]::IsNullOrEmpty($AllJobs)) {
+    Write-Output "No Jobs found to set Post Script for."
+    Start-Sleep 10
+    exit
+}
+# Command that Veeam needs to use for the Post Script action.
+$VeeamPowershellCommand = "Powershell.exe -File C:\VeeamScripts\VeeamDiscordNotifications\Bootstrap.ps1"
+
+# Run foreach loop for all found jobs
+foreach ($JobName in $AllJobs) {
+    $Job = Get-VBRJob -Name $JobName -ea silentlyContinue -WarningAction silentlyContinue
+    # Get all required options/parameters for specific job
+    $JobOptions = $Job.GetOptions()
+    $JobScriptCommand = $JobOptions.JobScriptCommand
+    $PostScriptEnabled = $JobScriptCommand.PostScriptEnabled
+    $PostScriptCommandLine = $JobScriptCommand.PostScriptCommandLine
+    # Different actions whether Post Script is already enabled. If yes we ask to modify it, if not we ask to enable & set it.
+    if ($PostScriptEnabled -eq 'True') 
+    {
+        $OverWriteCurrentCL = Read-Host "`r`nThere is already a Post Script set for $JobName`r`nScript: $PostScriptCommandLine`r`nDo you want to overwrite it? Y/N"
```

Comment:
`` `r`` isn't necessary since `` `n`` alone achieves the same result.
```suggestion
        $OverWriteCurrentCL = Read-Host "`nThere is already a Post Script set for $JobName`nScript: $PostScriptCommandLine`nDo you want to overwrite it? Y/N"
```